CMakeLists.txt

```diff
@@ -182,12 +172,28 @@ set(QATERIAL_SRCS
 source_group(TREE "${QATERIAL_SOURCE_DIR}" FILES ${QATERIAL_SRCS})
 source_group(TREE "${QATERIAL_ICONS_OUTPUT_FOLDER}/Qaterial" FILES ${QATERIAL_GEN_SRCS})
 
-set(QATERIAL_SRCS ${QATERIAL_SRCS} ${QATERIAL_GEN_SRCS})
+set(QATERIAL_SRCS ${QATERIAL_SRCS})
 
 # ───── QATERIAL TARGET ─────
 
+qt_policy(SET QTP0004 NEW)
```

Comment:
This is neat since Qt6.8, thank you for the discovery I missed it. I guess it won't change much in here, but I have some projects that may be affected so I can remove lots of CMake code.